application-licensing-test/application-licensing-test-example/pom.xml

```diff
@@ -56,4 +57,17 @@
       <scope>runtime</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.xwiki.commons</groupId>
+        <artifactId>xwiki-commons-tool-xar-plugin</artifactId>
+        <configuration>
+          <visibleTechnicalPages>
+            <visibleTechnicalPage>.*/Example/WebHome\.xml</visibleTechnicalPage>
```

Comment:
This page is used in functional tests where we access it directly (i.e. we don't expect it to appear in search results, in the live table or in the page tree), so I think it can be hidden. Alternatively you can set its default locale to English (so that it's not viewed as technical page).